.github/workflows/coderabbit-review-ntfy.yml

```
@@ -0,0 +1,24 @@
+  notify:
+    if: >
+      github.event.issue.pull_request &&
+      github.event.comment.user.login == 'coderabbitai[bot]' &&
+      (contains(github.event.comment.body, '') || contains(github.event.comment.body, 'Actionable comments posted'))
+    runs-on: ubuntu-latest
+
+    steps:
+        - name: Send ntfy notification
+          env:
+            NTFY_TOPIC: ${{ secrets.NTFY_TOPIC }}
+          run: |
```

actionlint (1.7.4)

"github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)